server/controllers/userController.js

```diff
@@ -50,3 +51,52 @@ exports.updateUserInfo = async function (req, res) {
     return res.status(400).json({ massage: err });
   }
 };
+
+// Update name, avaterUrl and timeZone
+exports.signUpUser = async function (req, res) {
+  const name = req.body.name;
+  const email = req.body.email;
+  const avatarUrl = req.body.avatarUrl;
+  let userId = "";
+
+  // Check user has account
+  const existUser = await Users.find({ email: email }, { email: 1 });
+  console.log(existUser.length);
+  if (existUser.length > 0) {
+    return res.status(200).json({ massage: "Already Exist User Account" });
+  }
+
+  try {
+    // Create new User
+    const user = new Users({
+      name: name,
+      email: email,
+      avatarUrl: avatarUrl,
+      timeZone: "America/Toronto", // Default
+      calendarUrl: "",
+    });
+    userId = user._id;
+    user.calendarUrl = user.createUrl();
+    user.save();
+  } catch (err) {
+    return res.status(400).json({ massage: err });
+  }
+
+  // Create default meeting
```

Comment:
This is great I was looking exactly for this for my Onboarding integration. Actually everything might work out for the frontend. 
I don't think we need to create default meeting for new users. It would be done by meeting controller when they click new meeting on the dashboard. 
So when the user gets to dashboard for the first time we will display a card that say you have no new meetings please create one.
Also meeting id will be unique in meeting model so you cant use 1 as default for every user. This  I am not 100% sure but pretty certain.

Reply:
I thought when the user signed up meanwhile new meeting also create.
Let me check

Reply:
in one of my tickets, i did recommend creating a default meeting for each new user

this is like a default meeting meant for quick onboarding :) 

you're right about the meeting id though, it should be unique for each user